azurepipelines/e2e_test/scenarios/ubuntu-18.04-amd64/vm_setup/devicesetup.py

```diff
@@ -34,8 +34,10 @@ def main():
 
     if (len(connectionString) == 0):
         print_error("Failed to create the device in the IotHub")
-        exit(1)
+        sys.exit(1)
 
+    print(str(len(connectionString)))
+    print(connectionString)
```

Comment:
will some bot in the pipeline flag that secrets are being logged?  If that is likely, then this in all the devicesetup.py should be omitted.

Reply:
Yes but best to remove it anyway